client/components/backup-retention-management/constants.ts

```diff
@@ -0,0 +1,11 @@
+export const RETENTION_OPTIONS = {
+	RETENTION_DAYS_7: 7,
```

Comment:
Is there a good reason to have those declared in an object? They seem pretty simple and used only in one place.

Reply:
Double-checked and no. It was an attempt to declare what are the options that we are going to use. I removed this and instead, added the `RetentionPeriod` from `calypso/state/rewind/retention/types` to validate that we use valid values.

Addressed in #73303